<a href="https://colab.research.google.com/github/SeungJoo03/BP_05_Ch3/blob/main/Python___3___Report.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

용자로부터 두 개의 정수를 받°圖 정수의 합 정수의 차 정수의 곱 졍수의 펑균, 큰
수 작은 수를 계산하여 화면에 출럭하는 프로그랩을 작셩하라 파이썬이 제공하는 내
장 함수 max(x, y), min(x, y)을 사용해보자.

In [1]:
x = int(input("x : "))
y = int(input("y : "))

sum = x + y
diff = x - y
mul = x * y
aver = sum / 2

print("두 수의 합 :", sum)
print("두 수의 차 :", diff)
print("두 수의 곱:", mul)
print("큰 수 :", max(x,y))
print("작은 수 :", min(x,y))

x : 10
y : 20
두 수의 합 : 30
두 수의 차 : -10
두 수의 곱: 200
큰 수 : 20
작은 수 : 10


기둥의 부피를 계산하는 프로그랩을 작성해보자. 원기둥의 부피는 다옴과 같이 계산한다


In [2]:
r = int(input("r :"))
h = int(input("h :"))

v = 3.141592 * r ** 2 * h

print("원기둥의 부피 :", v)

r :10
h :100
원기둥의 부피 : 31415.92


용지로부터 정수를 받。내 정수의 자러수의 합을 계산하는 프로그랩을 작성하여 보자. 예릍 들어서 사용자가 1234를 입럭하였다면 1+2+3+4를계산하면 된다 나머지 연산자와 정수 나눗셈 연산자와 정수 나눗셈 연산자 //를 적극적으로 사용해보자

In [3]:
x = int(input("정수를 입력하시오 : "))

one = x % 10
x = x // 10
ten = x % 10
x = x // 10
hun = x % 10
tho = x // 10
sum = one + ten + hun + tho

print("자리수의 합 :", sum)

정수를 입력하시오 : 1234
자리수의 합 : 10


용자로부터 투 점의 좌표 (xl, yl)과 (x2, y2)를 입럭받아서 두 점 사이의 거러를 계산하는 프로그랩을 작셩해보자 스크렵트 모드로 작성하라 거리는 다옴 식으로 계산한다.

In [4]:
x1 = int(input("x1 : "))
y1 = int(input("y1 : "))
x2 = int(input("x2 : "))
y2 = int(input("y2 : "))

dis = ((x1 - x2) ** 2 + (y1 - y2) ** 2) ** 0.5

print("두 점 사이의 거리 : ", dis)

x1 : 0
y1 : 0
x2 : 100
y2 : 100
두 점 사이의 거리 :  141.4213562373095


1번분졔에서 계산한거러가맞는지, 터틀그
래퍽으로 확인해보자. 거북이를 읜쪽으로 45
도회전하여 141만큼 젼진시킨다. 다시 거북이
를 (0, 0)으로 이동하고 0도를 가러키게 한 후
에 100만큼 전진하고 왼쪽으로 90도 회전하여
100만큼 전진한다. 화면에 그러진 직션이 일치
하는가?


사용자로부터 두점을업럭받。圖 터틀그래퍽을이용하여 두점을연결하는직선을그
린다 직션의 끝점에 직션의 길이를 계산하여 츨럭해보자

이썬에서 timeO을호출하면 1970닌 1월 1일 이후홀러은 전체 초가반환된다 이것을
적절하게 나누거나 나머지를 구해서 현재 시간의 시와 분을 계산할 수 있올까?


직이는 물체의 운동에너지를 계산해보자. 물체의 에너지를 계산하는 식은 1/2*무게*
속도*2이다.